src/qibo/models/circuit.py

```diff
@@ -8,6 +8,7 @@
 from qibo import gates
 from qibo.config import raise_error
 from qibo.gates.abstract import Gate
+from qibo.imaging.mpldrawer import MPLDrawer
```

Comment:
As said before, avoid importing the drawing module in `circuit.py`.

The motivation is that `mpldrawer` is a UI module, more or less an extension. While `circuit` is one of the core elements.
So, it should definitely go the other way round. And importing without a similar criterion largely increases the chances of circular imports.

Reply:
Thanks @alecandido !!! this is done!